Core/Wrappers/Damages/Damage.cs

```diff
@@ -386,6 +506,15 @@ private static double CalculatePhysicalDamage(this Obj_AI_Base source, Obj_AI_Ba
                         armorPenetrationPercent = 0.3d;
                         break;
                 }
+                if (target is Obj_AI_Minion)
+                {
+                    amount *= 1.25;
```

Comment:
https://github.com/LeagueSharp/LeagueSharp.Common/commit/b1fafbfaa36b4df2e4f37ba32b6a731a879d271e
I base on this, if sth wrong then ask Honda ^^